label_studio/core/settings/base.py

```diff
@@ -891,3 +892,7 @@ def collect_versions_dummy(**kwargs):
 # Data Manager
 # Max number of users to display in the Data Manager in Annotators/Reviewers/Comment Authors, etc
 DM_MAX_USERS_TO_DISPLAY = int(get_env('DM_MAX_USERS_TO_DISPLAY', 10))
+
+# Base FSM (Finite State Machine) Configuration for Label Studio
+FSM_CACHE_TTL = 300  # Cache TTL in seconds (5 minutes)
+FSM_AUTO_CREATE_STATES = False
```

Comment:
Dont see any usages of this here or in LSE , is this future planned work?

Reply:
Yea I took this out entirely, or so I thought 😆. Nice catch